util/src/main/java/com/google/cloud/hadoop/util/AsyncWriteChannelOptions.java

```diff
@@ -20,45 +20,70 @@
 import com.google.auto.value.AutoValue;
 import com.google.common.flogger.GoogleLogger;
 
-/** Options for the {@link AbstractGoogleAsyncWriteChannel}. */
+/**
```

Comment:
Please revert this formatting - it was already formatted w/ https://github.com/google/google-java-format